types/glob-to-regexp/glob-to-regexp-tests.ts

```diff
@@ -0,0 +1,26 @@
+import * as globToRegExp from 'glob-to-regexp';
```

Comment:
An ES6 module namespace object cannot be invoked as a function. Instead use `import globToRegExp = require('glob-to-regexp');`.

Reply:
Thank you for reviewing! I fixed both.